README.md

```diff
@@ -1 +1,40 @@
-# team05-to-DO
\ No newline at end of file
+# Project 2: Todo App
+
+## project descrption
+This project is a demonstration of a to-do app that allows the user to manage their tasks and have them saved in their browser using local storage.
+
+## User Stories
+ - The user will be able to add to their to-do list and save it to their local storage
+ - The user will be able to edit the task of their choice
+ - The user will be able to remove the task of their choice
+ - The user can check a task and mark it as done
+ - The user will be able to see their list if they reopen their browser window
+ - The user will be able to toggle dark mode on and off and save that choice for next sessions
+## Live link
+
+
+## How to get the project on your machine
+you can easily do this by cloning the repository in a folder at your machine, and we will walk you through those steps one by one :)
+
+1. click the Code button and copy the link to the repository
+
+![](https://i.imgur.com/wFg8Gce.png)
+
+2. open a folder and open git bash there
+
+![](https://i.imgur.com/MRAMJnu.png)
+
+3.open the command line and paste the repository link as the following: 
+`git clone https://github.com/GSG-CF04/team05-to-DO`
+
+![](https://i.imgur.com/uoiFNqR.png)
+ 
+ 4. after entering that, you should be able to view a folder with all the files of this project 🎉✨
+
+![](https://i.imgur.com/u63zdxL.png)
+
+
+
+
+## Team members
+This project was created by [Aseel AbuSahmoud](https://github.com/AseelL), [Husam Kamal](https://github.com/husamkamal) and [Muhanna Atalla](https://github.com/muhannaAtalla20) under the supervision and guidance of [Sara J.](https://github.com/sara219)
```

Comment:
add the team members as a list, it will look much better 

- [Aseel AbuSahmoud](https://github.com/AseelL)
- [Husam Kamal](https://github.com/husamkamal)
- [Muhanna Atalla](https://github.com/muhannaAtalla20)

Reply:
done